docker/Dockerfile

```diff
@@ -35,9 +39,9 @@ ENV PATH=/home/work/incubator-superset/superset/bin:$PATH \
     PYTHONPATH=./superset/:$PYTHONPATH
 
 COPY docker-init.sh /usr/local/bin/
-RUN chmod +x /usr/local/bin/docker-init.sh
+
+RUN chmod +x /usr/local/bin
 
 EXPOSE 8088
 
 ENTRYPOINT ["docker-init.sh"]
```

Comment:
Have we tested with a full and fresh docker build.  I suspect below issues

- The dependencies are not updated in setup.py
-  Not all files are copied to /usr/local/bin
- Working DIR set before it was created